server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -330,6 +330,6 @@ public void accessTokenResponse() throws Throwable {
 
     @Then("^the attachment is named \"([^\"]*)\"$")
     public void assertContentDisposition(String name) throws IOException {
-        assertThat(response.getHeaders("Content-Disposition")).extracting(Header::toString).containsExactly("Content-Disposition: attachment; filename=\"" + name + "\"");
+        assertThat(response.getHeaders("Content-Disposition")).extracting(Header::toString).containsExactly("Content-Disposition: attachment");
```

Comment:
Ok :
- the 2 scenarios have been changed because we could not check the name anymore because it has previously been removed from the content-disposition header. 
- we add a line to check the value of the content-disposition header